.github/workflows/shipjs-schedule-prepare.yml

```diff
@@ -0,0 +1,26 @@
+name: Ship js Schedule Prepare
+on:
+  schedule:
+    - cron: '0 9 * * 2'
```

Comment:
You can change the cron expression if there's any preferred time/date.
Currently it's every Tuesday 02:00 UTC.